package.json

```diff
@@ -29,6 +29,7 @@
     "@opentelemetry/instrumentation": "0.28.0",
     "@opentelemetry/instrumentation-express": "0.28.0",
     "@opentelemetry/instrumentation-http": "0.28.0",
+    "@opentelemetry/instrumentation-mongodb": "0.29.0",
```

Comment:
We should be use the same version as the other instrumentations: 0.28.0. Mixing release ls is dangerous.

Reply:
hi :) 0.28 support only mongo >=3.3<4 while 0.29 support also 4